### ISSUE TITLE
Brenk and Dundee filters are identical

### ISSUE THREAD

**Issue by j-adamczyk** [rdkit/rdkit#7797]
**Describe the bug**

[Brenk filter](https://github.com/rdkit/rdkit/blob/e4f4644a89d6446ddebda0bf396fa4335324c41c/Code/GraphMol/FilterCatalog/brenk.in) and [Dundee filter](https://github.com/rdkit/rdkit/blob/e4f4644a89d6446ddebda0bf396fa4335324c41c/Code/GraphMol/FilterCatalog/chembl_dundee.in) are identical, just slightly differently formatted. They both come from paper ["Lessons Learnt from Assembling Screening Libraries for Drug Discovery for Neglected Diseases" R. Brenk et al.](https://doi.org/10.1002/cmdc.200700139). This is also described in [these presentation slides](https://www.soci.org/-/media/files/conference-downloads/2011/designing-safer-medicines-in-discvery-mar-2011/francis_atkinson_presentation.ashx).

Naming confusion probably arose from the fact that Ruth Brenk is the first author, while the filter was developed at the University of Dundee. AFAIK the "Brenk filter" is more commonly known, so I think that Dundee filter should be removed. This is mainly in [Filters.cpp](https://github.com/rdkit/rdkit/blob/e4f4644a89d6446ddebda0bf396fa4335324c41c/Code/GraphMol/FilterCatalog/Filters.cpp#L375) file.

**Comment by bp-kelley** [comment]
Thanks for the info, we can just make the change to return BRENK when Dundee is requested so we don't have two sets of filters and make a note in the docs.  I forget where the original files came from in any case, so it's good to consolidate these.


```
case FilterCatalogParams::BRENK:
case FilterCatalogParams::DUNDEE:
      return NUM_BRENK;
```

**Comment by j-adamczyk** [comment]
@bp-kelley Dundee comes from https://github.com/PatWalters/rd_filters. Making a note is definitely a good idea. Maybe also mark one as deprecated alias?

**Comment by bp-kelley** [comment]
FilterCatalog predates rd_filters by a good bit.  Looks like it was used as the ChemBL 23 filters and we just never noticed they were the same.

**Comment by j-adamczyk** [comment]
You're right, I found it here: https://chembl.blogspot.com/2015/02/chembl-20-released.html. They all it Dundee, not Brenk, so this is probably a better default name after all.

**Comment by j-adamczyk** [comment]
Still relevant. If there is a consensus that Brenk is left and Dundee can be removed, I can try making a PR for this.

**Comment by greglandrum** [comment]
> Still relevant. If there is a consensus that Brenk is left and Dundee can be removed, I can try making a PR for this.

How about we deprecate both and just include one called "Brenk/Dundee" so that it's clear to people expecting either one?

**Comment by bp-kelley** [comment]
I think making a new filter will cause unnecessary workflow issues, I think changing BRENK and DUNDEE to point to the same filter set is the best option here.  We can change the name in the filter metadata and add a comment indicating that these are the same sets.

**Comment by bp-kelley** [comment]
I.e. 
```
  enum FilterCatalogs {
    PAINS_A = (1u << 1),
    PAINS_B = (1u << 2),
    PAINS_C = (1u << 3),
    PAINS = PAINS_A | PAINS_B | PAINS_C,

    BRENK = (1u << 4),
    NIH = (1u << 5),
    ZINC = (1u << 6),

    CHEMBL_Glaxo = (1u << 7),
    CHEMBL_Dundee = BRENK,
```

**Comment by j-adamczyk** [comment]
I agree with keeping just one, preferably Brenk, to avoid breaking changes in [scikit-fingerprints](https://github.com/scikit-fingerprints/scikit-fingerprints), it would probably also affect [medchem](https://github.com/datamol-io/medchem). So pointing Dundee to Brenk sounds like the best option to me.
